### ISSUE TITLE
generation failed when go in "modules" mode

### ISSUE THREAD

**Issue by sbezverk** [openconfig/gribi#4]
With recent go, when go modules are on by default, generating proto files are failing.
```
make
cd /home/user/gribi/proto/gribi_aft/enums && go generate
go: cannot find main module, but found .git/config in /home/user/gribi
	to create a module there, run:
	cd ../../.. && go mod init
make: *** [Makefile:5: generate] Error 1
```
Would the maintainers of this repo consider refactoring PR enabling the code generation with go modules?

**Comment by robshakir** [comment]
Absolutely, let me look at updating this today. Thanks.

**Comment by sbezverk** [comment]
Ok, I thought to do it, but if you do it, even better ;) thank you for quick reply..

**Comment by robshakir** [comment]
PTAL at #5 -- this should cover this fix.

**Comment by robshakir** [comment]
Fixed in #5.
